mudlet-mapper.xml

```diff
@@ -5390,7 +5421,29 @@ speedWalkCounter = 0
 mmp.speedWalk = mmp.speedWalk or {}
 mmp.speedWalkPath = mmp.speedWalkPath or {}
 mmp.speedWalkDir = mmp.speedWalkDir or {}
-local newversion = "developer"
+mmp.lagtable = {
+    [1] = {
+        description = "Normal, default level.",
+        time = 0.5
+    },
+    [2] = {
+        description = "Decent, but slightly laggy.",
+        time = 1
+    },
+    [3] = {
+        description = "Noticably laggy with occasional spikes.",
+        time = 2
+    },
+    [4] = {
+        description = "Bad. Terrible. Terribad.",
+        time = 5
+    },
+    [5] = {
+        description = "Carrier Pigeon",
+        time = 10
+    }
+}
+local newversion = "21.1.1"
```

Comment:
```suggestion
local newversion = "developer"
```

An automatic script releases every Tuesday (or Thursday, don't remember which) and it auto-sets the version.

Reply:
Ahh, wasn't sure. I was changing from the latest release I had, I think. will set it to developer and update.